src/NServiceBus.Gateway/Receiving/SingleCallChannelReceiver.cs

```diff
@@ -39,7 +39,7 @@ public void Dispose()
 
         void DisposeManaged()
         {
-            channelReceiver?.Dispose();
+            channelReceiver?.Stop();
```

Comment:
I think this will never be called since the outer class doesn't implement `IDisposable`. Should you just stop the `SingleCallChannelReceiver` which then stops the inner receiver?